test_scripts/API/GetWayPoints/001_GetWayPoints_success_flow.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/25
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/GetWayPoints/detailed_docs/TRS/embedded_navi/GetWayPoints_TRS.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid and allowed by Policies GetWayPoints_request to SDL
+-- SDL must:
+-- 1) Transfer GetWayPoints_request to HMI
+-- 2) Respond with <resultCode> received from HMI to mobile application
+-- 3) Provide the requested parameters at the same order as received from HMI 
+--    to mobile application (in case of successfull response)
+
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonGetWayPoints = require('test_scripts/API/GetWayPoints/commonGetWayPoints')
+
+local response ={}
+  response.wayPoints =
+  {{
+      coordinate =
+      {
+        latitudeDegrees = 1.1,
+        longitudeDegrees = 1.1
+      },
+      locationName = "Hotel",
+      addressLines =
+      {
+        "Hotel Bora",
+        "Hotel 5 stars"
+      },
+      locationDescription = "VIP Hotel",
+      phoneNumber = "Phone39300434",
+      locationImage =
+      {
+        value ="icon.png",
+        imageType ="DYNAMIC",
+      },
+      searchAddress =
+      {
+        countryName = "countryName",
+        countryCode = "countryCode",
+        postalCode = "postalCode",
+        administrativeArea = "administrativeArea",
+        subAdministrativeArea = "subAdministrativeArea",
+        locality = "locality",
+        subLocality = "subLocality",
+        thoroughfare = "thoroughfare",
+        subThoroughfare = "subThoroughfare"
+      }
+  } }
+
+
+--[[ Local Functions ]]
+local function GetWayPoints(self)
+    local params = { 
+      wayPointType = "ALL",
+      appID = commonGetWayPoints.getHMIAppId() 
```

Comment:
@LitvinenkoIra "appID" is not a parameter for request (App->SDL). But it's a parameter for request (SDL->HMI)